topo/fot/forch/behaviors.yaml

```diff
@@ -8,6 +8,7 @@ device_mac_behaviors:
   "9a:02:57:1e:8f:03":
     segment: INFRA
     auto_sequestering: enabled
```

Comment:
somewhat begs the question around what does "auto_sequestering" mean -- because there's all sorts of "auto" -- not something we need to change/address now, more just pontificating.

Reply:
the package is readily available.